src/main/java/gameframework/game/GameConfiguration.java

```diff
@@ -13,20 +13,44 @@
 
 public class GameConfiguration {
 
+	protected int nbRows;
```

Comment:
I think all of these variables could be `final` as they probably won't change during execution. Please fix and rebase your commit.